src/index.js

```diff
@@ -0,0 +1,5 @@
+import requireIndex from "requireindex"
+
+export default {
+  rules: requireIndex(`${__dirname}/rules`),
```

Comment:
There's a linting error for `__dirname`, you can fix that by adding this to the `.eslintrc.js`:
```
env: {
  node: true
}
```